strings-csharp/HowToAddSeparatorsInThousandsPlaceForANumber/Tests/Tests.cs

```diff
@@ -0,0 +1,109 @@
+using System.Globalization;
+
+namespace Tests
+{
+    public class Tests
+    {
+        private readonly double _balance = 2154002.535;
+        private readonly CultureInfo _englishCultureInfo = new("en-US");
+        private readonly CultureInfo _spanishCultureInfo = new("es-ES");
+
+        // String.Format method
+        [Fact]
+        public void GivenNumericValue_WhenStringFormatIsUsed_ShouldReturnExpectedStringValue()
+        {
+            var message = String.Format("Your account balance is {0}", _balance);
```

Comment:
@jshergal 
Would it be better here to test the Program.cs method or leave it this way?